modules/govuk_jenkins/templates/jobs/publishing_api_check_validity.yaml.erb

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: Publishing_API_Check_Validity
+    display-name: Publishing_API_Check_Validity
+    project-type: freestyle
+    description: "This job checks the data within the Publishing API for any invalid data"
+    logrotate:
+      artifactNumToKeep: 10
+    publishers:
+       - project: run-rake-task
+         predefined-parameters: |
+          TARGET_APPLICATION=publishing-api
+          MACHINE_CLASS=backend
+          RAKE_TASK=duplicate_content_items:all
+    triggers:
+       - timed: 'H 5 * * *'
```

Comment:
Yup - I was aiming for once each day at around 5am. 

Although I realise now that at that time integration is powered off - so I guess I should probably set it later unless it remains pending until integration is switched on (it will go out to prod eventually)